docs/.vuepress/config.js

```diff
@@ -6,27 +6,46 @@ module.exports = (_ctx) => ({
     '/': {
       lang: 'en-US',
       title: 'MetaMask Docs',
-      description: 'Developer documentation for the MetaMask Ethereum wallet'
-    }
+      description: 'Developer documentation for the MetaMask Ethereum wallet',
+    },
   },
 
   head: [
     ['link', { rel: 'icon', href: `/metamask-fox.svg` }],
     ['link', { rel: 'manifest', href: '/manifest.json' }],
     ['meta', { name: 'theme-color', content: '#3eaf7c' }],
     ['meta', { name: 'apple-mobile-web-app-capable', content: 'yes' }],
-    ['meta', { name: 'apple-mobile-web-app-status-bar-style', content: 'black' }],
-    ['link', { rel: 'apple-touch-icon', href: `/icons/apple-touch-icon-152x152.png` }],
-    ['link', { rel: 'mask-icon', href: '/icons/safari-pinned-tab.svg', color: '#3eaf7c' }],
-    ['meta', { name: 'msapplication-TileImage', content: '/icons/msapplication-icon-144x144.png' }],
-    ['meta', { name: 'msapplication-TileColor', content: '#000000' }]
+    [
```

Comment:
Lots of churn in this PR. Was `prettier` run on this or something? We can move this to another PR.

Reply:
I submitted a separate `lint:fix` PR but this file wasn't touched by the linter.

Reply:
Addressed here: https://github.com/MetaMask/metamask-docs/pull/346